src/api/app/assets/stylesheets/webui/application/style.scss

```diff
@@ -412,3 +412,10 @@ div.dataTables_wrapper:after {
 #subheader #breadcrump {
   line-height: 1.3em;
 }
+
+// flash messages
+.flash-message {
+  p {
+      margin: 10px;
+  }
+}
```

Comment:
Well, it actually was styled but only for the `id` case. Since I cannot display more than one flash message, I needed to introduce classes but I think I'll gonna make a new PR and move flash messages to a class instead of `id` to get rid of duplicated code...